nixos/modules/hardware/corectrl.nix

```diff
@@ -0,0 +1,40 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.corectrl;
+in {
+
+  options.programs.corectrl = {
+    enable = mkEnableOption ''
+      A tool to overclock amd graphics cards and processors.
+      Add your user to the corectrl group to run corectrl without it asking for your password
+    '';
+
```

Comment:
```suggestion
```

Reply:
i'll setup my editor to remove those automatically